docs/api-docs/provider-alibaba.md

```diff
@@ -4,4 +4,4 @@ toc: true
 weight: 402
 indent: true
 redirect_to: https://doc.crds.dev/github.com/crossplane/provider-alibaba
----
```

Comment:
Not exactly sure what is happening with these changes, but seems fine 🤷‍♂️

Reply:
I think they're just adding newlines where none were before - I ran a script to find and add trailing newlines to all files that didn't have them.